src/components/ListItem/ListItem.tsx

```diff
@@ -46,14 +69,21 @@ export const ListItem: React.FC<IListItemProps> = ({
     );
 
     return (
-        <li className={classnames(cssListItem)}
+        <li
+            className={classnames(cssListItem)}
+            tabIndex={tabIndex}
             {...props}
         >
             {
-            iconStart &&
+                iconStart &&
                 <iconStart.type {...iconStart.props} size="small" className={classnames('moonstone-listItem_iconStart')}/>
```

Comment:
It seems the className `moonstone-listItem_iconStart` render `undefined` . You can see it in the `ListItem/Icon + Text` storybook example.

Reply:
Ah yes, I saw this as per above. I've made a fix for now.